readme.md

```diff
@@ -120,7 +120,9 @@ The Microsoft Graph SDK is open for contribution. To contribute to this project,
 
 ## 6. Supported Java versions
 
-The Microsoft Graph SDK for Java library is supported at runtime for Java 8+ and [Android API revision 21](http://source.android.com/source/build-numbers.html) and greater.
+The Microsoft Graph SDK for Java library is supported at runtime for Java 8+ and [Android API revision 26](http://source.android.com/source/build-numbers.html) and greater.
```

Comment:
nit: either
- remove `+` in `Java 8+` or
- remove `greater` and add `+` to android version.

Reply:
```suggestion
The Microsoft Graph SDK for Java library is supported at runtime for Java 8 and [Android API revision 26](http://source.android.com/source/build-numbers.html) or greater.
```